packages/augur-core/source/contracts/utility/RedeemStake.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity 0.5.4;
+
+import 'ROOT/reporting/IDisputeWindow.sol';
+import 'ROOT/reporting/IReportingParticipant.sol';
+
+
+/**
+ * @title Redeem Stake
+ * @notice A Utility contract with no state for redeeming many redeemable bonds or tokens into the ystsem in on TX
+ */
+contract RedeemStake {
+
+    /**
+     * @notice Redeems stake for multiple dispute bonds or Participation Tokens
+     * @param _reportingParticipants Winning Initial Reporter or Dispute Crowdsourcer bonds the msg sender has stake in
+     * @param _disputeWindows Dispute Windows (Participation Tokens) the msg sender has tokens for
+     * @return Bool True
+     */
+    function redeemStake(IReportingParticipant[] memory _reportingParticipants, IDisputeWindow[] memory _disputeWindows) public returns (bool) {
```

Comment:
so token holder needs to redeem. Is there a need for other addresses to redeem on behalf of other account?

Reply:
No real need IMO. Holding forever with these bonds doesn't lock up anyone's funds but the holders